src/screens/FeedScreen/Footer.js

```diff
@@ -103,12 +103,12 @@ const styles = StyleSheet.create({
 
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
-    return '#00ADB5';
+    return COLORS.holyTosca;
   }
   if (totalVote < 0) {
-    return '#FF2E63';
+    return COLORS.red;
   }
-  return '#C4C4C4';
+  return COLORS.gray1;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `handleTextCountColor` has been updated to return color constants from the `COLORS` object instead of hard-coded color values. This change improves maintainability by centralizing color definitions in one place (`utils/theme`). As long as the `COLORS` constants are correctly defined, this change should not introduce any issues.
```diff
-    return '#00ADB5';
+    return COLORS.holyTosca;
...
-    return '#FF2E63';
+    return COLORS.red;
...
-  return '#C4C4C4';
+  return COLORS.gray1;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->